scripts/decontaminate.py

```diff
@@ -1,3 +1,3 @@
-#!/usr/bin/python
+#!/usr/bin/env python
```

Comment:
We should be able to remove the scripts directory, and instead have setup.py make the scripts for us.  If we make this switch, Python will wire up the correct interpreter for the scripts it creates.

For an example, see the `console_scripts` argument here:
https://github.com/kylebittinger/unassigner/blob/master/setup.py